main/src/main/java/org/mobilitydata/gtfsvalidator/notice/UnusedShapeNotice.java

```diff
@@ -0,0 +1,16 @@
+package org.mobilitydata.gtfsvalidator.notice;
```

Comment:
What are your thoughts on moving the `notice` package to the `core` module? I think it would be useful to have all the notice definitions `core` package so it could be used as a dependency in another project. Same with `validator` package that's currently in `main`. `main` package could be reduced to the CLI.

Reply:
There is already a `notice` package in `core`. It contains the most basic notices such as `EmptyFileNotice`, `ForeignKeyError` that do not depend on particular schema.

Let's discuss in another comment whether to move the majority of `main` to `core`.